src/pip/_internal/cli/cmdoptions.py

```diff
@@ -903,7 +903,8 @@ def _handle_config_settings(
     dest="root_user_action",
     default="warn",
     choices=["warn", "ignore"],
-    help="Action if pip is run as a root user. By default, a warning message is shown.",
+    help="Action if pip is run as a root user ('warn' or 'ignore'). "
+    "By default, a warning message is shown.",
```

Comment:
```suggestion
    help="Action if pip is run as a root user [warn, ignore]. "
    "(default: 'warn')",
```

I think this is the format we use in most options

Reply:
Good idea to make this more consistent. It's now similar to:
`[...] Specify which mechanism to use [disabled, import, subprocess] (default: disabled)`
and
`Specify whether the progress bar should be used [on, off] (default: on)`